puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/smart-proxy-plugin-pull-request.yaml

```diff
@@ -0,0 +1,55 @@
+- job-template:
+    name: 'smart-proxy-{plugin}-pull-request'
+    concurrent: true
+    project-type: matrix
+    logrotate:
+      daysToKeep: 45
+      numToKeep: -1
+    properties:
+      - github:
+          url: 'https://github.com/theforeman/smart_proxy_{plugin}'
+    scm:
+      - git:
+          url: 'https://github.com/theforeman/smart_proxy_{plugin}'
+          wipe-workspace: true
+          prune: true
+          branches:
+            - '${{ghprbActualCommit}}'
+          refspec: '+refs/pull/${{ghprbPullId}}/*:refs/remotes/origin/pr/${{ghprbPullId}}/*'
+    triggers:
+      - github_pr:
+          context: 'smart_proxy_{plugin}'
+    wrappers:
+      - timeout:
+          type: absolute
+          abort: true
+          timeout: 60
+          write-description: 'Build timed out (after {{0}} minutes). Marking the build as aborted.'
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - '2.2'
```

Comment:
Smart proxy still runs under 2.0 on EL7. Don't think we have a 2.2 platform left though.

Reply:
oops! I removed the wrong numbers!

Reply:
updated